components/html-block/html-block.js

```diff
@@ -129,14 +140,14 @@ class HtmlBlock extends LitElement {
 			const fragment = template ? document.importNode(template.content, true) : null;
 			if (fragment) {
 
-				const hasMath = !!fragment.querySelector('math');
+				const hasMathML = !!fragment.querySelector('math');
 
 				const temp = document.createElement('div');
 				temp.appendChild(fragment);
 				const fragmentHTML = temp.innerHTML;
 
-				if (hasMath) {
-					await loadMathJax();
+				if (hasMathML || /\$\$|\\\(/.test(fragmentHTML)) {
```

Comment:
Right now we're only supporting the `$$` and `\(` delimiters because that's what Carlton currently needs. If others feel strongly about it, I can expand this to include the `\[` delimiter as well.